bcgov/HOUS-permit-portal
# UI cleanup round5

## Description

Fixing some buggy stuff that was missed last time.

Also fixes for: 
- https://hous-bssb.atlassian.net/browse/BPHH-643
- https://hous-bssb.atlassian.net/browse/BPHH-1029
- https://hous-bssb.atlassian.net/browse/BPHH-1030
- https://hous-bssb.atlassian.net/browse/BPHH-1031
- https://hous-bssb.atlassian.net/browse/BPHH-732

## What type of PR is this? (check all applicable)


## Related Tickets & Documents



## Steps to QA

- Look at a submitted application
- Look at the contact card at bottom of a "About LG" page
- Look at Requirements Library table view:
-- should see a "Edit" link per requirement block
-- should see a yellow label "Has Elective(s)" for ones that do
-- clicking into one to "edit requirement", should see a yellow label that says "Elective" for the form field that is.
- With the Edit Requirement Block modal still open, click button "Add requirement", and the drawer should be back to being clickable now.
- nav dropdown menu is a bit more tidied up now
- as a super admin editing a template and you want to reorder the sections, now the UI matches mockups more where the right side of the screen is greyed out and the requirement blocks are not interact-able. (same thing for when a super admin is editing a blank template)
- on landing page ,  put in an address to see your jurisdiction - the big blue thing is now clickable

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?
